src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
 // write your code here
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise((resolve) => {
+  logo.addEventListener('click', resolve);
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(reject, 3000);
+});
+
+function onSuccess() {
+  const successMessage = document.createElement('div');
+
+  successMessage.className = 'message';
+  successMessage.textContent = 'Promise was resolved!';
+  document.body.append(successMessage);
+}
+
+function onError() {
+  const errorMessage = document.createElement('div');
+
+  errorMessage.className = 'message error-message';
+  errorMessage.textContent = 'Promise was rejected!';
+  document.body.append(errorMessage);
+}
+
+promise1.then(onSuccess).catch(onError);
+promise2.then(onSuccess).catch(onError);
```

Comment:
Currently, `promise1` and `promise2` are handled independently. If the intention is to handle them together, consider using `Promise.race` or `Promise.all` to manage their outcomes collectively. This way, you can determine which promise settles first or wait for both to settle, depending on the desired behavior.